src/scripts/main.js

```diff
@@ -1,3 +1,84 @@
 'use strict';
 
-// write code here
+const mainTable = document.querySelector('.field');
+const tbody = mainTable.querySelector('tbody');
+
+const addColumnButton = document.querySelector('.append-column');
+const addRowButton = document.querySelector('.append-row');
+const removeColumnButton = document.querySelector('.remove-column');
+const removeRowButton = document.querySelector('.remove-row');
+
+addColumnButton.addEventListener('click', addColumn);
+addRowButton.addEventListener('click', addRow);
+removeColumnButton.addEventListener('click', removeColumn);
+removeRowButton.addEventListener('click', removeRow);
+
+function addColumn(evenD) {
```

Comment:
The parameter name 'evenD' seems to be a typo. It should be 'event' to follow convention.